packages/manager/modules/vps/src/upscale/upscale.service.js

```diff
@@ -0,0 +1,56 @@
+import isString from 'lodash/isString';
+
+import TranslateService from '../../../core/src/translate/translate.service';
```

Comment:
you are supposed to import external dependencies like that, what if we move folders ?

you must do ```import TranslateService from '@ovh-ux/manager-core/src/translate/translate.service';```

Reply:
You're actually the one who told me to do that a few months ago 😅 

You're right though

Reply:
Ok, done, but this requires me to add `/* eslint-disable import/extensions */`.

Also, I see that we do this many times in the manager, shall we just disable this rule?

Reply:
It's not better though, because it means you have to know how the files are structured in the `manager-core` module, the best would be to `export` `TranslationService` instead